x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -1491,6 +1495,32 @@ export const tasks: TelemetryTask[] = [
       };
     },
   },
+  {
+    name: 'custom_dashboards',
+    executor: async ({ savedObjectsClient }) => {
+      const response = await savedObjectsClient.find<SavedApmCustomDashboard>({
+        type: APM_CUSTOM_DASHBOARDS_SAVED_OBJECT_TYPE,
+        page: 1,
+        perPage: 500,
+        sortField: 'updated_at',
+        sortOrder: 'desc',
+        namespaces: ['*'],
+      });
+
+      const kueryNodes = response.saved_objects.map(
+        ({ attributes: { kuery } }) => fromKueryExpression(kuery ?? '')
+      );
```

Comment:
@TinaHeiligers made a good point regarding the `size` and how this could be a perf issue. It we can perform aggregations on SOs (big if) we can do something like:

```ts
aggs: {
 terms:  {
  field: 'saved_objects.attributes.kuery'
 }
}
```

This way we'll only get unique `kuery` values, and not the full SO docs.

Reply:
According to the doc, https://www.elastic.co/guide/en/kibana/current/saved-objects-api-find.html I'll give it a try. However, is experimental

Reply:
Thanks! Experimental is fine for telemetry 👍